src/services/items/interfaces/item-task-manager.ts

```diff
@@ -35,11 +35,17 @@ export interface ItemTaskManager<A extends Actor = Actor> {
     itemTask: Task<A, Item>,
     options: { parentId?: string; shouldCopyTags?: boolean },
   ): Task<A, unknown>[];
+  createGetChildrenTask(
+    member: Actor,
+    { item, ordered }: { item?: Item; ordered?: boolean },
+  ): Task<Actor, unknown>;
   createGetChildrenTaskSequence(
     actor: Actor,
     itemId: string,
     ordered?: boolean,
   ): Task<A, unknown>[];
   createGetOwnTask(actor: Actor): Task<A, Item[]>;
-  createGetSharedWithTask(actor: Actor, input: GetItemsSharedWithTaskInputType): Task<A, Item[]>;
+  createGetSharedWithTask(actor: Actor, input: { permissions?: string[] }): Task<A, Item[]>;
+  createGetDescendantsTask(member: Actor, input?: { item: Item }): Task<A, Item[]>;
```

Comment:
So you can use `getInput` later. Otherwise you cannot "dynamically" input the value. (ie task1 get the item from the id, you input the result in `getDescendantsTask`)